darts/datasets/__init__.py

```diff
@@ -648,3 +648,168 @@ def _to_multi_series(self, series: pd.DataFrame) -> List[TimeSeries]:
+    % UNWEIGHTED ILI: Combined state-specific data of patients visit to healthcare providers for ILI reported each week unweighted by state population
+    AGE 0-4: Number of patients between 0 and 4 years of age
+    AGE 25-49: Number of patients between 25 and 49 years of age
+    AGE 25-64: Number of patients between 25 and 64 years of age
+    AGE 5-24: Number of patients between 5 and 24 years of age
+    AGE 50-64: Number of patients between 50 and 64 years of age
+    AGE 65: Number of patients above (>=65) 65 years of age
+    ILITOTAL: Total number of ILI patients. For this system, ILI is defined as fever (temperature of 100°F [37.8°C] or greater) and a cough and/or a sore throat
+    NUM. OF PROVIDERS: Number of outpatient healthcare providers
+    TOTAL PATIENTS: Total number of patients
+
+    References
+    ----------
+    .. [1] https://gis.cdc.gov/grasp/fluview/fluportaldashboard.html
+    .. [2] https://www.cdc.gov/flu/weekly/overview.htm#Outpatient
+    .. [3] https://arxiv.org/pdf/2205.13504.pdf
+    """
+
+    def __init__(self):
+        super().__init__(
+            metadata=DatasetLoaderMetadata(
+                "ILINet.csv",
+                uri="https://raw.githubusercontent.com/unit8co/darts/Improvement/Add_new_datasets_617/datasets/ILINet.csv",
+                hash="c9cbd6cc0a92b21cd95bec2706212d8d",
+                header_time="DATE",
+                format_time="%Y-%m-%d",
+                freq="W",
+            )
+        )
+
+
+class ExchangeRateDataset(DatasetLoaderCSV):
+    """
+    The collection of the daily exchange rates of eight foreign countries including Australia, British, Canada,
+    Switzerland, China, Japan, New Zealand and Singapore ranging from 1990 to 2010.
+
+    References
+    ----------
+    .. [1] https://github.com/laiguokun/multivariate-time-series-data
+    """
+
+    def __init__(self, multivariate: bool = True):
+        """
+        Parameters
+        ----------
+        multivariate: bool
+            Whether to return a single multivariate timeseries - if False returns a list of univariate TimeSeries. Default is True.
+        """
+        super().__init__(
+            metadata=DatasetLoaderMetadata(
+                "exchange_rate.csv",
+                uri="https://raw.githubusercontent.com/unit8co/darts/Improvement/Add_new_datasets_617/datasets/exchange_rate.csv",
+                hash="9219e9a03eb54c6e40d7eb1c9b3b6f7c",
+                header_time="Date",
+                format_time="%Y-%m-%d",
+                freq="D",
+                multivariate=multivariate,
+            )
+        )
+
+    def _to_multi_series(self, series: pd.DataFrame) -> List[TimeSeries]:
+        """
+        Load the ExchangeRateDataset dataset as a list of univariate timeseries, one for each country.
+        """
+        return [
+            series[label]
```

Comment:
This seems to be returning a pandas Series, we need to return a `TimeSeries`. You can probably simply do `TimeSeries.from_series(series[label])`.